integration/test/Test/Teams.hs

```diff
@@ -0,0 +1,102 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2024 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module Test.Teams where
+
+import API.Brig
+import API.BrigInternal (createUser, getInvitationCode, refreshIndex)
+import API.Galley (getTeamMembers)
+import API.GalleyInternal (setTeamFeatureStatus)
+import Control.Monad.Codensity (Codensity (runCodensity))
+import Control.Monad.Reader (asks)
+import Data.String.Conversions (cs)
+import Network.HTTP.Types.URI (parseQuery)
+import Network.URI (URI (uriQuery), parseURI)
+import Notifications (isUserUpdatedNotif)
+import SetupHelpers
+import Testlib.JSON
+import Testlib.Prelude
+import Testlib.ResourcePool (acquireResources)
+
+testInvitePersonalUserToTeam :: (HasCallStack) => App ()
+testInvitePersonalUserToTeam = do
+  resourcePool <- asks (.resourcePool)
+  runCodensity (acquireResources 1 resourcePool) $ \[testBackend] -> do
+    let domain = testBackend.berDomain
+    (owner, tid, tm) <- runCodensity (startDynamicBackend testBackend def) $ \_ -> do
+      (owner, tid, tm : _) <- createTeam domain 2
+      pure (owner, tid, tm)
+
+    runCodensity
+      ( startDynamicBackend
+          testBackend
+          (def {galleyCfg = setField "settings.exposeInvitationURLsTeamAllowlist" [tid]})
```

Comment:
we've done this a few times in the past, and usually within a few months somebody needed this to go from yaml to cql...  i think we should make this a table right away.

Reply:
I think this is not really related to the feature, but would only make the test nicer in this case.